contracts/modules/v2/StakingPool.sol

```diff
@@ -354,4 +367,53 @@ contract StakingPool is ERC20 {
 
   }
 
+  /* VIEWS */
+
+  /* ========== PRICE CALCULATION ========== */
+
+  function getUsedCapacity(uint productId) public view returns (uint) {
+    return 0;
+  }
+
+  function getCapacity(uint productId, uint capacityFactor) public view returns (uint) {
+    return 0;
+  }
+
+  function calculatePremium(uint pricePercentage, uint coverAmount, uint period) public pure returns (uint) {
+    return pricePercentage * coverAmount / MAX_PRICE_PERCENTAGE * period / 365 days;
+  }
+
+  function calculatePrice(
+    uint amount,
+    uint basePrice,
+    uint activeCover,
+    uint capacity
+  ) public pure returns (uint) {
+
+    return (calculatePriceIntegralAtPoint(
```

Comment:
I think I am out of context here. I'm pretty sure there's a doc with the formulas that might be worth adding here as comment to help people that read the code understand what is going on.